sharding/nodesSetup.go

```diff
@@ -320,41 +327,45 @@ func (ns *NodesSetup) InitialNodesInfoForShard(shardId uint32) ([]nodesCoordinat
 
 // NumberOfShards returns the calculated number of shards
 func (ns *NodesSetup) NumberOfShards() uint32 {
-	return ns.nrOfShards
+	return ns.numberOfShards
 }
 
 // MinNumberOfNodes returns the minimum number of nodes
 func (ns *NodesSetup) MinNumberOfNodes() uint32 {
-	return ns.nrOfShards*ns.MinNodesPerShard + ns.MetaChainMinNodes
+	return ns.computeMinNumberOfNodes()
```

Comment:
do we still need 2 methods for MinNumberOfNodes?
I would remove the second internal one.

Reply:
removed the internal one